apps/web/src/features/coaches/utils/index.ts

```
@@ -0,0 +1,16 @@
+  const type = contractType.toLowerCase();
+
+  if (type === "w2" || type === "w-2") {
+    // Full-time employee - primary/default style
+    return "bg-primary/10 text-primary";
+  } else if (type === "hourly" || type.includes("hour")) {
+    // Hourly contractor - secondary style
+    return "bg-secondary text-secondary-foreground";
+  } else {
+    // Default/Unknown - muted style
+    return "bg-muted text-muted-foreground";
+  }
```

Biome (2.1.2)

[error] This else clause can be omitted because previous branches break early.

Safe fix: Omit the else clause.


(lint/style/noUselessElse)